file_paths.md

```diff
@@ -25,13 +25,23 @@ The files annotated by an '*' indicate that they are created by package installa
     /etc/puppetlabs/code *                # :codedir                     contents moved from :confdir
         environments *                    # :environmentpath
           production *
+            hieradata *
             environment.conf *
             manifests *
             modules *
         hiera.yaml *                      # :hiera_config
-        hieradata *                       # n/a
         modules *                         # user modulepath
 
+    /etc/puppetlabs/code-staging          # staging directory            n/a
```

Comment:
(I guess we already do this with facts.d, so maybe I'm just saying too littel too late. )